singolo/singolo.html

```diff
@@ -0,0 +1,161 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Yauhen Vaznishchyk">
+  <title>Singolo</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet"> 
+</head>
+<body>
+  <header class="header">
+    <div class="wrapper">
+      <div class="header-content">
+        <div class="logo">
+          <h1>SINGOLO</h1>
+          <span class="logo-symbol">*</span>
+        </div>
+        <nav>
+          <div class="hamburger-menu">
+            <input id="menu-toggle" type="checkbox"/>
+            <label class="menu-btn" for="menu-toggle">
+              <span></span>
+            </label>
+          <ul class="menu-box">
+            <li class="menu-item">SINGOLO<span>*</span></li>
+            <li><a class="menu-item" href="#">HOME</a></li>
+            <li><a class="menu-item" href="#services">SERVICES</a></li>
+            <li><a class="menu-item" href="#portfolio">PORTFOLIO</a></li>
+            <li><a class="menu-item" href="#">ABOUT</a></li>
+            <li><a class="menu-item" href="#">CONTACT</a></li>
+          </ul>
+        </div>
+        </nav>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="slider">
+      <div class="wrapper">
+        <div class="slider-content">
+          <img class="icon-previous-page" src="images/slides/arrow-left.svg" alt="previous page">
+          <img class="phone-vertical" src="images/slides/phone-vertical.png"
+            alt="vertical iphone with green image on the screen">
+          <img class="phone-horizontal" src="images/slides/phone-horizontal.png"
+            alt="horizontal iphone with blue image on the screen">
+          <img class="icon-next-page" src="images/slides/arrow-right.svg" alt="next page">
+        </div>
+      </div>
+    </section>
+    <section class="services" id="services">
+      <div class="wrapper">
+        <h2>Our Services</h2>
+        <p>Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum. Duis mollis, non commodo luctus, nisi
+          erat
+          porttitor ligula, eget lacinia odio sem nec elit. Fusce dapibus, tellus ac cursus commodo, tortor mauris
+          condimentum nibh.</p>
```

Comment:
If a text in a tag does not fit in one line, it would be better to arrange it as follows:
```suggestion
        <p>
          Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum. Duis mollis, non commodo luctus, nisi
          erat porttitor ligula, eget lacinia odio sem nec elit. Fusce dapibus, tellus ac cursus commodo, tortor mauris
          condimentum nibh.
        </p>
```